apps/protocol-api/src/prisma/migrations/20221216154936_add_favorite_membership_status/migration.sql

```diff
@@ -3,4 +3,4 @@ ALTER TABLE "GuildUser" ADD COLUMN     "favorite" BOOLEAN NOT NULL DEFAULT false
 ADD COLUMN     "membership_status_id" INTEGER;
 
 -- AddForeignKey
-ALTER TABLE "GuildUser" ADD CONSTRAINT "GuildUser_membership_status_id_fkey" FOREIGN KEY ("membership_status_id") REFERENCES "GuildMembershipStatus"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+ALTER TABLE "GuildUser" ADD CONSTRAINT "GuildUser_membership_status_id_fkey" FOREIGN KEY ("membership_status_id") REFERENCES "GuildMembershipStatus"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
This update in order to stop Prisma from removing then re-adding the fk key in a distinct migration.

Reply:
Yes.

Reply:
Actually, I remembered that in the first [migration](https://github.com/Govrn-HQ/govrn-monorepo/pull/340), I manually changed it to `RESTRICT` instead of `SET NULL` to align with some of other migrations.

Reply:
Can we undo this? the migrations have already been applied on prod and staging. we aren't going to unapply and then reapply them. If we want to make this change we should modify the .prisma file.